cdap-app-fabric/src/main/java/io/cdap/cdap/security/impersonation/DefaultUGIProvider.java

```diff
@@ -50,15 +62,19 @@
   private final LocationFactory locationFactory;
   private final File tempDir;
   private final NamespaceQueryAdmin namespaceQueryAdmin;
+  protected final Store store;
```

Comment:
Why the field needs to be protected instead of private? Generally not a good practice.